entity-framework/core/providers/ibm/index.md

```diff
@@ -22,15 +19,15 @@ This database provider allows Entity Framework Core to be used with IBM Data Ser
 
 To work with IBM Data Server on Windows, install the [IBM.EntityFrameworkCore NuGet package](https://www.nuget.org/packages/IBM.EntityFrameworkCore).
 
-````text
+``` console
```

Comment:
Is there a different one for text that contains the prompt (`PM>`) vs ones that don't? The GitHub Pages one has two.